packages/next/cli/next-lint.ts

```diff
@@ -61,7 +61,8 @@ const nextLint: cliCommand = async (argv) => {
     '--max-warnings': Number,
     '--no-inline-config': Boolean,
     '--report-unused-disable-directives': String,
-    '--cache': Boolean,
+    '--cache': Boolean, // Although cache is enabled by default, this dummy flag still exists to not cause any breaking changes
```

Comment:
Many users likely opt-in to caching today using `next lint --cache`. I kept `--cache` as a dummy flag so it doesn't cause any breaking changes.